content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -104,40 +104,65 @@ OpenTelemetry's [Agent Configuration][11] page describes additional properties t
 [9]: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/
 [10]: https://search.maven.org/search?q=a:opentelemetry-javaagent-jaxws-2.0-cxf-3.0
 [11]: https://opentelemetry.io/docs/zero-code/java/agent/configuration/
+[16]: /opentelemetry/interoperability/environment_variable_support
 
 {{% /tab %}}
 
-<!-- {{% tab "Python" %}}
+{{% tab "Python" %}}
 
 ## Compatibility requirements
 
+The Datadog Python SDK supports library [instrumentations][13] using the Opentelemetry Python Trace API.
+
+OpenTelemetry provides an [example][14] for instrumenting a sample application.
+
 ## Setup
 
-## Configuration
+To use Opentelemetry integrations with the Datadog Python SDK, perform the following steps:
 
-{{% /tab %}} -->
+ 1. Follow the instructions in the [OpenTelemetry API][15] section in the Datadog Python library docs. 
+ 2. Follow the steps for instrumenting your service with your chosen `opentelemetry-python-contrib` library.
 
-<!-- {{% tab "Ruby" %}}
+
+[13]: https://github.com/open-telemetry/opentelemetry-python-contrib/tree/0a231e57f9722e6101194c6b38695addf23ab950/instrumentation#readme
```

Comment:
If you're not worried about the content changing, you could reference the main branch like this. Otherwise, ignore this comment 🙂 

```suggestion
[13]: https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation#readme
```